src/generator/app.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var express = require('express');
+var app = express();
+
+var generator = require('./backend/generator.js');
+
+app.set('port', (process.env.PORT || 5000));
+
+// Use the www folder as static frontend
+app.use('/', express.static(__dirname + '/www'));
+
+app.all('/generate', function (req, res) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2708752417/issues/source?bid=3233857&fileBranchId=3342834#l12)